### ISSUE TITLE
Thank you

### ISSUE THREAD

**Issue by ExposedCat** [max-wickham/GWarden#1]
Not sure if this is still supported but I love the initiative. Thank you for your work

**Comment by Froggy232** [comment]
Yes, this initiative is very nice, thanks to the author even if it's not maintained anymore.
Is it the case?
Thanks you!
